packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -280,8 +289,16 @@ export class AppManagementClient implements DeveloperPlatformClient {
       )
   }
 
-  async templateSpecifications(_appId: string): Promise<ExtensionTemplate[]> {
-    return stubbedExtensionTemplates()
+  async templateSpecifications({organizationId}: MinimalAppIdentifiers): Promise<ExtensionTemplate[]> {
+    const allExtensions = await Promise.all(templateJsonUrls.map(async (url) => {
+      const response = await fetch(url)
+      return response.json() as Promise<GatedExtensionTemplate[]>
+    })).then(templates2DArray => templates2DArray.flat(1))
```

Comment:
If there are duplicate definitions across the two jsons (and the fact there are two repos is obviously a historical mistake), which wins? Is that an error?

Reply:
I think it should be logged at least. It points at an internal team misconfiguring the manifests.

Reply:
We'll investigate the possibility of having just 1 file.

Reply:
We have concluded that we'll use 1 file, and the order in the file will be the same as the order displayed in the CLI. (The CLI will still group the options, so that can override order, but we should try and keep each group together in the file for readability.)